src/app/app.config.ts

```diff
@@ -25,7 +25,12 @@ export const appConfig: ApplicationConfig = {
     importProvidersFrom(provideFirestore(() => getFirestore())),
     provideAnimationsAsync(),
     provideHttpClient(),
-    provideStore(),
+    provideStore(
+      {},
+      {
+        metaReducers: [debug],
```

Comment:
This needs to be that other one with the `isDev() ? [debug] : []` otherwise you spam the console in prod